installing/installing_aws/installing-aws-customizations.adoc

```diff
@@ -17,30 +17,16 @@ The scope of the {product-title} installation configurations is intentionally na
-xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
-processes.
-* xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[Configure an AWS account]
-to host the cluster.
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[configured an AWS account] to host the cluster.
 +
 [IMPORTANT]
 ====
-If you have an AWS profile stored on your computer, it must not use a temporary
-session token that you generated while using a multi-factor authentication
-device. The cluster continues to use your current AWS credentials to create
-AWS resources for the entire life of the cluster, so you must use long-lived
-credentials. To generate appropriate keys, see
-link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing Access Keys for IAM Users]
-in the AWS documentation. You can supply the keys when you run the installation
-program.
+If you have an AWS profile stored on your computer, it must not use a temporary session token that you generated while using a multi-factor authentication device. The cluster continues to use your current AWS credentials to create AWS resources for the entire life of the cluster, so you must use long-lived credentials. To generate appropriate keys, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Managing Access Keys for IAM Users] in the AWS documentation. You can supply the keys when you run the installation program.
 ====
-* If you use a firewall, you must
-xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
-* If you do not allow the system to manage identity and access management (IAM),
-then a cluster administrator can
-xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam-aws[manually
-create and maintain IAM credentials]. Manual mode can also be used in
-environments where the cloud IAM APIs are not reachable.
+* If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
+* If you do not allow the system to manage identity and access management (IAM), then a cluster administrator can xref:../../installing/installing_aws/manually-creating-iam.adoc#manually-creating-iam-aws[manually create and maintain IAM credentials]. Manual mode can also be used in environments where the cloud IAM APIs are not reachable.
```

Comment:
I know this is existing text. Do you know what is meant by the _system_? Will the users?

Reply:
This is a good question. My understanding is that it refers to the OpenShift Cloud Credential Operator (CCO) managing cloud provider credentials as Kubernetes custom resource definitions. As outlined, for example, in [Manually creating IAM for AWS](https://deploy-preview-33681--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/manually-creating-iam.html#manually-creating-iam-aws), you can configure the CCO to suit your security requirements by setting values for the `credentialsMode` parameter in the `install-config.yaml` file.

Given that the prerequisite is used widely in the Installer book, I will reword it in this PR so that the underlying reasons for manual credential managment are clearer.

Reply:
I have updated each instance of this bullet point as follows. The link changes depending on the platform subsection:

"* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can [manually create and maintain IAM credentials](https://deploy-preview-33681--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/manually-creating-iam.html#manually-creating-iam-aws)."